worker/worker.py

```diff
@@ -681,6 +681,13 @@ def locked_by_others(lock_file, require_valid=True):
         return True
     return False
 
+def utcoffset():
```

Comment:
https://docs.python.org/3/library/datetime.html?highlight=utcoffset#datetime.datetime.utcoffset exists?

Reply:
I tried this but all my attempts failed. E.g.
```
>>> print(datetime.datetime.now().utcoffset())
None
```
It's a bit like java programming. You search the docs until you have finally found a class that seems to do what you want, and then it turns out to be an abstract class.

Reply:
Actually my implementation is wrong (for offsets that are not multiple of hours).

Reply:
I wouldn't try to implement that.... however the python one requires  this:

datetime.tzinfo
The object passed as the tzinfo argument to the datetime constructor, or None if none was passed.

so somehow one needs to init datetime correctly.

Reply:
Yes, if one can get hold of a `tzinfo` object describing the local timezone. It seems that ought to be possible...

Reply:
https://stackoverflow.com/questions/32402502/how-to-change-the-time-zone-in-python-logging/47104004